google-cloud-resourcemanager/README.md

```diff
@@ -186,11 +186,7 @@ Policy policy = project.getPolicy();
 // Add a viewer
 Policy.Builder modifiedPolicy = policy.toBuilder();
 Identity newViewer = Identity.user("<insert user's email address here>");
-if (policy.bindings().containsKey(Role.viewer())) {
-  modifiedPolicy.addIdentity(Role.viewer(), newViewer);
-} else {
-  modifiedPolicy.addBinding(Role.viewer(), newViewer);
-}
+modifiedPolicy.addIdentity(Role.of("roles/viewer"), newViewer);
```

Comment:
Nit, for a snippet I would prefer the shorter/simpler version of `Role.of("viewer")`